cfme/tests/cli/test_appliance_console.py

```diff
@@ -1433,4 +1429,9 @@ def test_ap_failed_dbconfig_status():
             2.
             3. The command "echo $?" should be non-zero
     """
-    pass
+    appliance = temp_appliance_preconfig_funcscope
+    command = ("/opt/rh/cfme-gemset/bin/appliance_console_cli -i -b /dev/sdz -S"
+               " -d ${db_name} -U ${db_root} -p '${db_pass}'")
+    result = appliance.ssh_client.run_command(command, timeout=30)
+    assert not result.success, ('DB configuration should fail because used disk "/dev/sdz" '
+                                'which will not be available')
```

Comment:
Step 3. of testSteps specifies `Check command execution status "echo $?"`. Why has this check not been included?

Reply:
`result.success`  will give True if command's return value `0 `so we do not need
 to do extra `echo $?`

Reply:
Ah that's correct, my mistake!